Editor/CKEditor/CKEditor.php

```diff
@@ -82,16 +82,16 @@ public function getVars()
     private function getDefaults()
     {
         return [
-            'barmode' => 'Standard',
+            'barmode' => 'Extended',
             'height' => '200',
             'resizemode' => 'resize',
             'resizeminheight' => '250',
             'resizemaxheight' => '3000',
             'growminheight' => '200',
             'growmaxheight' => '400',
-            'style_editor' => 'editors/ckeditor/css/contents.css',
+            'style_editor' => '',
```

Comment:
Removal on purpose?

Reply:
This default is doing nothing at all, the field itself is if you like to enter here site style sheet.